apps/arx/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    ca-certificates curl xdg-utils desktop-file-utils \
+    shared-mime-info libswt-gtk-4-java dbus-x11 at-spi2-core && \
+    curl -sSL "https://arx.deidentifier.org/?ddownload=2135" -o arxinstaller && \
```

Comment:
Did you try installing the Runnable Jar files? https://arx.deidentifier.org/downloads/

It feels overkill to download a full blown installer meant for humans and run it as _unattended_ when there is, what appears to be, a simpler way.

Reply:
Yes but it had some dependencies that i didn't found at the time so i went to the easiest solution.